### PR TITLE
make-based: Enhance common scripts

### PR DESCRIPTION
**Pull request by foxeng**
This fixes the equality test in the `install_lib` common function. In addition, it displays the usage message when and unknown command is specified to `do.sh`.

Hackathon team name: `ByteBusters`

### CODE REVIEW

File: make-based/include/common_command [hunk 1]
```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 
-if test $# -ne 1; then
+usage()
+{
     echo "Usage: $0 <command>" 1>&2
     echo "  command: setup configure build clean run run_debug remove" 1>&2
     exit 1
```
Comment (razvand) [inline make-based/include/common_command:7]: Do not place `exit 1` in `usage()` function. Leave it outside.

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @foxeng. See my comment. After that, we can upstream this.

Hackathon points: 5

File: make-based/include/common_command (hunk 1 above)
Comment (foxeng) [reply make-based/include/common_command]: Makes sense, fixed!

**Review by razvand — APPROVED** [review]
Thanks, @foxeng. Approved.

Hackathon-points: 5

**Pull request merged**